src/test/e2e/tests/specs.test.ts

```diff
@@ -176,6 +176,43 @@ describe('Finding a court or tribunal listing', () => {
     });
   });
 
+  describe('Following the subscription \'search\' by case reference path', () => {
+    const validSearchTerm = '487065515';
+    const invalidSearchTerm = 'dddd';
+    const expectedNumOfResults = 1;
+
+    it('should open the subscription add page', async () => {
+      await subscriptionAddPage.open('subscription-add');
+      expect(await subscriptionAddPage.getPageTitle()).toBe('How do you want to add a subscription?');
+    });
+
+    it('should see 4 radio buttons', async () => {
+      expect(await subscriptionAddPage.radioButtons).toBe(4);
+    });
+
+    it('should select \'By case reference number\' option and navigate to search urn page', async () => {
+      await subscriptionAddPage.selectCaseSearchRadio();
```

Comment:
please re-use selectOption function for radios we don't need separate serachRadios selector for each option as only the name of the radio is different

Reply:
this has been changed to use generic selectoption function.